### ISSUE TITLE
Decentralized block-lattice suggestion

### ISSUE THREAD

**Issue by alienc0der** [Silene0259/Decentralized-Login-Basics-silene#1]
Hello! I've read about what you are trying to achieve and wanted to let you know that [Zenon Network](https://github.com/zenon-network) has (almost) all the ingredients you need:

- dual-ledger system: beacon-chain for consensus and a block-lattice for settling transactions
- EdDSA (Ed25519) signature scheme
- is able to support VRFs and other functionalities via embedded contracts

The network also has an on-chain Accelerator that funds everything from protocol development to new use-cases like this one. If you're interested you can join the [community](https://my.znn.link/@LearnZenon) and submit a funding proposal [here](https://forum.zenon.org/).

**Comment by silenecapensis** [comment]
Interesting. I see it is written in Go and  I am a rust developer still trying to learn Go. I will look into it. Thanks for the suggestion.

**Comment by silenecapensis** [comment]
@alienc0der hey do you guys need any help. I have multiple skills I can apply to zenon. I have graphic design background, technical writing, documentation, web design, and programming. I would be happy to help out.

**Comment by alienc0der** [comment]
Yes, there is plenty of stuff that needs to be done. I think technical writing, documentation and programming are the most appropriate skills to apply with.

For example, we have an unfinished [Rust SDK](https://github.com/2bonahill/znn_sdk_rust) that needs to be done. This way you'll learn how the network works (it's a plus you already know how a block-lattice works), you'll get some funding for this and it will be easier for you to develop anything on top after that.

I'll close this issue and [contact you on email](https://atropinetears.github.io/contact.html) and we can move this conversation over there.

**Comment by silenecapensis** [comment]
Hey, could you email me at silene0259@gmail.com. I am interested in working on this with you. My old email doesnt work.

**Comment by silenecapensis** [comment]
Is it legitimate or just another scam? Network is down. Seemed like a good project but it doesnt have reliability plus there are some esoteric transactions. I am only interested in legitimate projects. Been in the game since 2011.

**Comment by alienc0der** [comment]
The mainnet is up and running since 2021 and the project is older than that. I encourage you to go down the rabbit hole..

> Network is down

Did you try to connect to a remote node? I suggest you [build from source](https://github.com/zenon-network/go-zenon?tab=readme-ov-file#building-from-source) or download the [pre-built binary](https://github.com/zenon-network/go-zenon/releases/latest) and run it by yourself.

> I am only interested in legitimate projects.

You can check their [Github](https://github.com/zenon-network) and see for yourself.

I've also [emailed you](mailto:AtropineTears@proton.me) to join the dev chat and ask questions.

**Comment by 0x3639** [comment]
@Silene0259 The project is not a scam.  We are a community of contributors from all over the world from all walks of life.   If you want to see the latest major contribution under way look at the Unikernel project.

https://forum.zenon.org/t/unikernel-z-a-unikernel-for-the-network-of-momentum/1871
https://github.com/EovE7Kj/unikernel-z 

The chain is not down.  You can check out the explorer here: [zenonhub.io](zenonhub.io)

You can also see our latest chain upgrades we are working on here:
https://forum.hypercore.one/t/ideas-for-dynamic-plasma/233

You can see some recent PRs here which will prepare us for dynamic plasma
https://github.com/zenon-network/go-zenon/pulls

Finally, Crypto Fish is working on [integrating PTLCs into a client](https://github.com/kinggorrin/znn_ptlc_use_cases_go).  The [embedded contract](https://github.com/zenon-network/go-zenon/pull/13) for `go-zenon` for PTLCs is done.

Check out the work flow an atomic swap with PTLCs
https://github.com/KingGorrin/znn_ptlc_use_cases_go/blob/main/docs/atomic-swaps-nom/atomic-swaps-nom.md

 This is not a scam.

**Comment by angeloajr** [comment]
> Is it legitimate or just another scam? Network is down. Seemed like a good project but it doesnt have reliability plus there are some esoteric transactions. I am only interested in legitimate projects. Been in the game since 2011.

Not a scam.  I'm currently building a dApp and running infrastructure on Zenon in parallel just so that we can use it in our application down the line.

Probably the most principled infrastructure I've ever seen.

**Comment by sugoibtc** [comment]
> Is it legitimate or just another scam? Network is down. Seemed like a good project but it doesnt have reliability plus there are some esoteric transactions. I am only interested in legitimate projects. Been in the game since 2011.

Glad to see you taking an interest in collaborating with our network!

I would like to reassure you that we are definitely not a scam. The project has been up and running without any downtime since the genesis of Alphanet (almost 3 years).

Zenon Network encourages open-source development and is both leaderless and fully community-driven. All of us are working hard and putting in countless hours to bring more attention to what we're trying to achieve. I'm hopeful that you're willing to join us.

**Comment by silenecapensis** [comment]
Hi, need any graphic design work done? Things like explaining Zenon and other things related to that. I can make infographics.

**Comment by 0x3639** [comment]
> Hi, need any graphic design work done? Things like explaining Zenon and other things related to that. I can make infographics.

Have you taken a look at the project?

**Comment by stark-of-zenon** [comment]
Infographics: I would love to see how much ZNN & QSR are locked in validators, staking, LP, etc. 

That, combined with token emissions info would paint a very nice picture.

**Comment by edgepillar** [comment]
@sileneundula I saw that you are interested in N&T, maybe this article will interest you

https://blog.zenon.info/narwhal-and-tusk-on-zenon-network/

**Comment by silenecapensis** [comment]
Is it still available for me to work on. I am learning Go and know Rust, Typescript, and some basic languages. I can also do graphic design and work on VMs. Who do I contact?

**Comment by alienc0der** [comment]
@sileneundula I suggest you should start with something fairly straightforward to accommodate with the network's concepts and architecture (which you should already be pretty familiar with as you're developing similar tech by yourself).

As I've already mentioned earlier in the thread, we have an unfinished [Rust SDK](https://github.com/2bonahill/znn_sdk_rust) that needs to be completed. Here is a list of all [Zenon SDK](https://forum.zenon.org/t/zenon-sdk-status/414) repos that you can take inspiration from. You can also check this [Go SDK](https://github.com/hypercore-one/go-zdk) that is better maintained.

Regarding the coordination with the community, I suggest you create an account on the [dev forum](https://forum.hypercore.one) and we'll help you with the onboarding into the dev channels.

**Comment by silenecapensis** [comment]
@alienc0der Will i be funded if I work on the Rust SDK? I will look into the network concepts and architecture.

**Comment by alienc0der** [comment]
Yes, everything development related is covered by an on-chain fund called [Accelerator-Z](https://zenonhub.io/accelerator-z). After the work is complete and evaluated, you can request funds from it. Can you join the [Matrix channel](https://forum.hypercore.one/t/matrix-chat-server-for-zenon-development/475) to continue the discussions? You can check this [page](https://www.zenon.chat) for more information on how to join.

**Comment by 0x3639** [comment]
Hello - please join this group on matrix: https://matrix.to/#/#general-dev:zenon.chat. You need a matrix.org account and you can join.  Please introduce yourself there and we are happy to show you around the project.  We are working on some game changing improvements to the project, but learning about the project by completing the Rust SDK would be a good first step if you are interested.

**Comment by silenecapensis** [comment]
I need a registration token.

**Comment by 0x3639** [comment]
Can you please send me a DM at @deeznnutz:zenon.chat on matrix?  I will invite your username to the group.

**Comment by sugoibtc** [comment]
Hey @sileneundula, you should be able to create an account without the token here: https://app.element.io/#/welcome

Afterwards you should be able to join our "general" room: https://matrix.to/#/#general:zenon.chat

**Comment by alienc0der** [comment]
I sent you an email at `AtropineTears@protonmail.com` with the registration token.

**Comment by silenecapensis** [comment]
Can you send it to silene0259@gmail.com @alienc0der

**Comment by silenecapensis** [comment]
Did Zenon really put it in the BTC chain during taproot?

**Comment by coinselor** [comment]
> Did Zenon really put it in the BTC chain during taproot?

Not only did they sign the taproot block with Zenon Network, they also added a cryptographic puzzle in the ASCII art.

So far the community hasn't been able to solve the puzzle.

Here's some more information about it:

https://forum.hypercore.one/t/12-seed-word-treasure-hunt/421

I've also made a write-up during my first attempt:

https://medium.com/@coinselor/zenons-treasure-hunt-the-missing-4-words-c99cbef8c65

Maybe you can help 😅

p.d: the community originally chose to burn the rewards in the address, but we [would be more than willing](https://forum.zenon.org/t/treasure-hunt-revival/1687) to award anyone that explains how it's solved.

**Comment by edgepillar** [comment]
> Did Zenon really put it in the BTC chain during taproot?

Also you can check these related articles;

https://www.zenon.org/en/blog/2023-11-14/two-years-since-zenon-network-engraved-in-bitcoins-taproot-upgrade
https://medium.com/@Zyler9985/zenon-network-i-the-green-pill-883e608727a

Zenon is really a very sophisticated initiative, if you want to discuss it with OGs and the community you are welcome;

https://www.zenon.chat/

**Comment by silenecapensis** [comment]
I read the Zenon hunt and was really interesting. Is the answer just the seven word hashing? Hashing is post-quantum secure, lets smart contracts run, and allows for blockchains to operate with integrity by keeping records of the blocks. I see it is encoded in base64. I am guessing the answer is the last four of the wallet.

@coinselor

**Comment by silenecapensis** [comment]
@alienc0der write me at silene0259@gmail.com with the invite code.

**Comment by coinselor** [comment]
Code sent 👍

**Comment by alienc0der** [comment]
> [@alienc0der](https://github.com/alienc0der) write me at [silene0259@gmail.com](mailto:silene0259@gmail.com) with the invite code.

Sent.

**Comment by edgepillar** [comment]
> > [@alienc0der](https://github.com/alienc0der) write me at [silene0259@gmail.com](mailto:silene0259@gmail.com) with the invite code.
> 
> Sent.

@sileneundula please note that invite codes expire in 24 hours
